packages/typespec-azure-portal-core/src/keys.ts

```diff
@@ -0,0 +1,10 @@
+import { createStateSymbol } from "./lib.js";
+
+export const PortalCoreKeys = {
```

Comment:
We actually have a better way to do that now by defining those in the `lib.ts` see http library for example https://github.com/microsoft/typespec/blob/ad74773b53e03700bfbb542467b67382308a77bb/packages/http/src/lib.ts#L133-L152

Reply:
fixed and deleted keys.ts